src/test/librbd/fsx.cc

```diff
@@ -2028,7 +2022,7 @@ main(int argc, char **argv)
 			break;
 		case 'l':
 			maxfilelen = getnum(optarg, &endp);
-			if (maxfilelen <= 0)
+			if (maxfilelen == 0)
```

Comment:
Looks like you didn't make this change, that I presume Coverity will otherwise complain about next time around? ;)